src/shared/test/bridge-test.js

```diff
@@ -1,5 +1,16 @@
-import Bridge from '../bridge';
-import { RPC } from '../frame-rpc';
+import { default as Bridge, $imports } from '../bridge';
```

Comment:
The Bridge tests were somewhat confusing because they mixed unit tests for the `Bridge` class alone, which mocked `RPC` in an unconventional way, with integration tests for Bridge + RPC that were not clearly demarcated. I pushed a commit that changes the tests in this module to all be unit tests which mock RPC using `$imports.$mock`, using the `FakeRPC` class below.